### PR TITLE
[Filter/TFLite] change interpreter to pointer

### PR DESCRIPTION
**Pull request by helloahn**
To make the interpreter shareable, need to change to a pointer first.

related to #3114

Signed-off-by: Hyoung Joo Ahn <hello.ahn@samsung.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc [hunk 1]
```diff
@@ -703,6 +691,22 @@ TFLiteInterpreter::cacheInOutTensorPtr ()
   return -EINVAL;
 }
 
+/**
+ * @brief	TFLiteCore constructor
+ */
+TFLiteCore::TFLiteCore ()
+{
+  interpreter = new TFLiteInterpreter ();
```
Comment (wooksong) [inline ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc:699]: ```TFLiteInterpreter``` looks like a struct without any constructors (default one would be invoked at this line).

Then, are there any reasons why we do not use ```TfLiteInterpreterCreate```?

https://github.com/tensorflow/tensorflow/blob/7ce597b5e8467cb662c9ae9a968e1ff10741f734/tensorflow/lite/c/c_api.cc#L131

File: ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc [hunk 2]
```diff
@@ -222,8 +223,8 @@ class TFLiteCore
   accl_hw accelerator;
   tflite_delegate_e delegate;
 
-  TFLiteInterpreter interpreter;
-  TFLiteInterpreter interpreter_sub;
+  TFLiteInterpreter *interpreter;
```
Comment (wooksong) [inline ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc:226]: Is this compatible with TF1.X?

File: ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc (hunk 1 above)
Comment (helloahn) [reply ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc:699]: The class `TFLiteInterpreter` is a different one you shared. It's a respective class.
PTAL https://github.com/nnstreamer/nnstreamer/blob/f1257d7c4d54699de88ce868f446a461084abb9e/ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc#L117

File: ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc (hunk 2 above)
Comment (helloahn) [reply ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc:226]: I try to test in various versions. Maybe I need to test more to open this PR.

**Comment by helloahn** [comment]
It'll be reopened after more test

**Pull request closed without merging**